deploy/deploy-wmagent.sh

```diff
@@ -260,6 +260,8 @@ set -e
 wget -nv -O deployment.zip --no-check-certificate https://github.com/dmwm/deployment/archive/$DEPLOY_TAG.zip
 unzip -q deployment.zip
 cd deployment-$DEPLOY_TAG
+# FIXME: comment this out before merging! Patch deployment configurations
+curl https://patch-diff.githubusercontent.com/raw/dmwm/deployment/pull/1088.patch | patch -p 1
```

Comment:
This will be fixed once the deployment PR gets merged.

Reply:
@amaltaro , why do we need to have patch numbers hard coded somewhere in the deployment script at the first place, or is it just a reminder for which patch needs to be added ?

Reply:
Yes, my comment above isn't very clear. What I meant is that I need to remove this line once my deployment PR 1088 gets merged.